.ibm/pipelines/resources/config_map/configmap-app-config-rhdh-rbac.yaml

```diff
@@ -88,6 +88,13 @@ data:
           rules:
             - allow: [User, Group]
       providers:
+        github:
```

Comment:
@nilgaar looks like your test will be running on showcase instance not on the showcase-rbac, if this is correct then this config update is not needed here, [configmap-app-config-rhdh.yaml](https://github.com/janus-idp/backstage-showcase/pull/1625/files#diff-d3959db965d3482fe2336650085090712166a0db3e7d9be769a42ff23eb70f5e) is enough

Reply:
fixed!! thank you for the insight!